src/testdrive/src/action.rs

```diff
@@ -92,6 +92,9 @@ pub struct Config {
     /// Backoff factor to use when retrying
     pub backoff_factor: f64,
 
+    /// `--arg-var` values provied to the cli
```

Comment:
If we're going to have these sorts of documentation comments, I'd really like to hold them to a higher standard:

```suggestion
    /// Arguments for the testdrive script to expose as variables.
    ///
    /// Each entry must be of the form `NAME=VALUE` and will be made available to
    /// the script as the variable named `arg.NAME`.
```

(as mentioned below requiring strings of `name=value` makes for a less ergonomic Rust API than the `BTreeMap<String, String>` you had before, but it's fine for now.)